### PR TITLE
Update Swift target with redesigned API and testing for ObjC

### PR DESCRIPTION
**Pull request by mdmathias**

### CODE REVIEW

**Comment by petea** [comment]
Might change the PR title to "Update Swift target"... as the Swift target already exists on `main`.

Should we resolve #199 before merging?  It would be nice to get the Objc name for AuthState in sync.

**Comment by mdmathias** [comment]
@petea updated PR title and happy to hold off merging while we resolve #199.

**Review by petea — APPROVED** [review]

**Comment by petea** [comment]
Discussed offline and we'll resolve #199 on `master`.

**Pull request merged**
